data/2024_25/seminare/ksp-praha/Z1_practical_end.yml

```diff
@@ -0,0 +1,16 @@
+name: KSP - Z1 - Deadline odovzdávania praktických úloh za tretinu bodov
+type: seminar
+sciences:
+  - inf
+date:
+  start: "2024-10-27"
+contestants:
+  min: null
```

Comment:
podľa mňa tam inak nechcete mať `null` potom sa to bude zobrazovať aj keď si nastavíš prvý ročník na základke.

Reply:
Technicky tam spodný limit nie je, ale ak to zavadzia tu v KK, tak sa môžeme o nejaký týždeň a pol pobaviť na orgovíkendovke o tom, čo tam dávať (a možno to rovno celé zautomatizujeme), zatiaľ by som nechal takto a prípadne znova otvorím a zmením...